### PR TITLE
Adds size support for ELF symbols.

### PR DESCRIPTION
**Pull request by Manouchehri**
Not sure if this breaks building on other platforms, but seems to work on Linux.

### CODE REVIEW

**Comment by jmis1337** [comment]
This would be very useful to port jemalloc parser to frida.

File: bindings/gumjs/gumv8module.cpp [hunk 1]
```diff
@@ -476,6 +476,7 @@ gum_emit_symbol (const GumSymbolDetails * details,
 
   _gum_v8_object_set_ascii (symbol, "name", details->name, core);
   _gum_v8_object_set_pointer (symbol, "address", details->address, core);
+  _gum_v8_object_set_uint(symbol, "size", details->size, core);
```
Comment (oleavr) [inline bindings/gumjs/gumv8module.cpp]:
```suggestion
  _gum_v8_object_set_uint (symbol, "size", details->size, core);
```

File: gum/backend-elf/gumelfmodule.h [hunk 2]
```diff
@@ -79,6 +79,7 @@ struct _GumElfSymbolDetails
   GumElfSymbolType type;
   GumElfSymbolBind bind;
   GumElfSectionHeaderIndex section_header_index;
+  gsize size;
```
Comment (oleavr) [inline gum/backend-elf/gumelfmodule.h]: Let's put this right after `address`.

**Comment by oleavr** [comment]
Actually, scratch that. Doesn't seem to be available on Darwin, at least not without parsing the DWARF. We can make this a Linux/ELF-only feature initially.

**Comment by oleavr** [comment]
Will take a quick stab at landing this.

**Comment by oleavr** [comment]
Landed in 2da7de17a125ca2881e795f4211d6b5da42eeb82. Thanks!

**Pull request closed without merging**